documentation/library_guide/extension_api.rst

```diff
@@ -313,3 +313,67 @@ Example of Range-based API usage
 
         copy(oneapi::dpl::execution::dpcpp_default, view, range_res);
     }
+
+Async API
+--------------------------
+
+Functions defined in the STL ``<algorithm>`` or ``<numeric>`` headers are traditionally blocking. oneDPL extends the functionality of C++17 parallel algorithms by providing asynchronous algorithm with non-blocking behavior. This experimental feature enables expressing concurrent control flow by building dependency chains and interleaving algorithm calls, as well as interoperability with SYCL kernels. 
```

Comment:
This should be updated so it does not all run on one long line. Please add the following updates (in **bold**):

Functions defined in the STL ``<algorithm>`` or ``<numeric>`` headers are traditionally blocking. oneDPL extends the functionality of C++17 parallel algorithms by providing **an** asynchronous algorithm with non-blocking behavior. This experimental feature enables **you to express a** concurrent control flow by building dependency chains and interleaving algorithm calls **and** interoperability with SYCL***** kernels. [DB question: is it SYLC kernels or DPC++ kernels?]

Reply:
Both are supported: SYCL and DPC++ kernels. Also other interfaces from oneAPI that return a `sycl::event`, for example: oneMKL.